src/AutoUI/index.tsx

```diff
@@ -189,7 +189,11 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 	}, [modelRaw]);
 
 	const [filters, setFilters] = React.useState<JSONSchema[]>([]);
-	const [sort, setSort] = React.useState<TableSortOptions<T> | null>(null);
+	const [sort, setSort] = React.useState<TableSortOptions<T> | null>(
+		(getFromLocalStorage(`${model.resource}__sort`) as
```

Comment:
Let's avoid I/O on every re-render
```suggestion
		() => (getFromLocalStorage(`${model.resource}__sort`) as
```